python/ray/actor.py

```diff
@@ -666,6 +676,7 @@ def _serialization_helper(self, ray_forking):
             _ray_actor_creation_dummy_object_id,
             "actor_method_cpus": self._ray_actor_method_cpus,
             "actor_job_id": self._ray_actor_job_id,
+            "session_index": self._ray_session_index,
```

Comment:
why do we need to serialize this?

Reply:
Otherwise I think there is no way to pass in `session_index` when forking.

Reply:
I mean, why do we need to pass `session_index` to the forked handle?
The forked handle might be used by a different worker. Passing `session_index` to a different worker doesn't make sense.